README.md

```diff
@@ -351,6 +361,7 @@ grafana_datasource { 'influxdb':
   grafana_url       => 'http://localhost:3000',
   grafana_user      => 'admin',
   grafana_password  => '5ecretPassw0rd',
+  grafana_api_path  => '/api'
```

Comment:
'/api' is the default value for grafana_api_path. I would add in a sub-path to show the use of the parameter, something like '/subpath/api' or '/grafana/api'